<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorDialog</class>
 <widget class="QDialog" name="MonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Sensors</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,50,0">
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="batt1Check">
          <property name="text">
           <string>Battery 1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="batt1Label">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>V</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="batt2Check">
          <property name="text">
           <string>Battery 2</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="batt2Label">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="batt2UnitLabel">
          <property name="text">
           <string>V</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="batt3Check">
          <property name="text">
           <string>Battery 3</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="batt3Label">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="batt3UnitLabel">
          <property name="text">
           <string>V</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="batt4Check">
          <property name="text">
           <string>Battery 4</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="batt4Label">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="batt4UnitLabel">
          <property name="text">
           <string>V</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="battPackCheck">
          <property name="text">
           <string>Battery Pack</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="battPackLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>V</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="powerCheck">
          <property name="styleSheet">
           <string notr="true">QCheckBox { background-color : red; color : blue; }</string>
          </property>
          <property name="text">
           <string>Power</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="powerLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>W</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="powerSetCheck">
          <property name="text">
           <string>Power Set</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="powerSetLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>W</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QCheckBox" name="tempCheck">
          <property name="text">
           <string>Temperature</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="tempLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLabel" name="tempUnit1Label">
          <property name="text">
           <string>°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QCheckBox" name="tempSetCheck">
          <property name="text">
           <string>Temperature Set</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="tempSetLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLabel" name="tempUnit2Label">
          <property name="text">
           <string>°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QCheckBox" name="outCurrCheck">
          <property name="text">
           <string>Output Current</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="outCurrLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>A</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QCheckBox" name="outVoltCheck">
          <property name="text">
           <string>Output Voltage</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="outVoltLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>V</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QCheckBox" name="coldResCheck">
          <property name="text">
           <string>Cold Resistance</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="coldResLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Ω</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QCheckBox" name="liveResCheck">
          <property name="text">
           <string>Live Resistance</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="liveResLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Ω</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QCheckBox" name="boardTempCheck">
          <property name="text">
           <string>Board Temperature</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="boardTempLabel">
          <property name="text">
           <string>?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <widget class="QLabel" name="tempUnit3Label">
          <property name="text">
           <string>°C</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pauseBtn">
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="recordBtn">
          <property name="text">
           <string>Record...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>X-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="xAxisCombo">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>5 s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20 s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>45 s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>60 s</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Y-Axis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="yAxisCombo">
            <property name="currentIndex">
             <number>4</number>
            </property>
            <item>
             <property name="text">
              <string>5 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100 %</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200 %</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
          <item>
           <widget class="QPushButton" name="puffBtn">
            <property name="text">
             <string>Puff...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="puffSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="showPuffsBoundCheck">
          <property name="text">
           <string>Show Puffs Boundaries</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showXAxisCheck">
          <property name="text">
           <string>Show X-Axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="showYAxisCheck">
          <property name="text">
           <string>Show Y-Axis</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="curveLayout">
     <item row="1" column="0">
      <widget class="QScrollBar" name="hScrollBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QScrollBar" name="vScrollBar">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="replace">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
